pkg/sfu/forwarder.go

```diff
@@ -1368,6 +1465,7 @@ func (f *Forwarder) GetTranslationParams(extPkt *buffer.ExtPacket, layer int32)
 	f.lock.Lock()
 	defer f.lock.Unlock()
 
+	// do not drop on publisher mute to enabled resume on publisher unmute without a key frame
```

Comment:
This is the bit that actually makes resume work faster, i. e. on pub mute, forwarding still happens. This ensures that streaming can continue on that layer on unmute. If the media stream track is not stopped on mute, it will send silence frames (black frames) which get forwarded and on resume there is no key frame, i.e publisher uses the black frame as reference.

Still need to do the dance of setting `parkedLayers` to ensure that there is some safety net on unmute to do a re-allocation just in case the layer does not resume on unmute.